x-pack/plugins/security_solution/public/resolver/view/panels/node_list.tsx

```diff
@@ -188,13 +219,30 @@ function NodeDetailLink({ name, item }: { name: string; item: ProcessTableView }
           )}
         </EuiBadge>
       ) : (
-        <>
+        <StyledButtonTextContainer>
           <CubeForProcess
             running={!isTerminated}
+            isOrigin={isOrigin}
             data-test-subj="resolver:node-list:node-link:icon"
           />
-          <span data-test-subj="resolver:node-list:node-link:title">{name}</span>
-        </>
+          <StyledProcessLabel color={descriptionText}>
+            {isOrigin && (
+              <div data-test-subj="resolver:node-list:node-link:origin" className="origin-label">
```

Comment:
I have the same request here re: https://styled-components.com/docs/advanced#referring-to-other-components